### PR TITLE
Add Hebrew translation

### PR DESCRIPTION
**Pull request by GreenLunar**

### CODE REVIEW

File: src/view/translation.js [hunk 1]
```diff
@@ -964,4 +964,69 @@ Candy.View.Translation = {
 
         'antiSpamMessage': 'Nespamujte prosím. Váš účet byl na chvilku zablokován.'
     }
+	'he' : {
```
Comment (benlangfeld) [inline src/view/translation.js]: Missing a comma before this block and a spurious one after, which is failing CI.

Comment (GreenLunar) [inline src/view/translation.js]: Thank you.

**Comment by benlangfeld** [comment]
Please squash for merge :)

**Comment by GreenLunar** [comment]
Do you want me to delete commit 3d3efecb9821d5102678fc0dea0df27b03c96ef0 with which I attempted to make a pull request?

**Comment by GreenLunar** [comment]
I do not  know how to squash.  I am closing this PR and I will send a new one.

**Pull request closed without merging**